src/style/list/conversations.less

```diff
@@ -292,28 +292,6 @@
   }
 }
 
-.conversations-settings-button {
-  .button-reset-default;
-  .button-icon-large;
-  .focus-border-radius;
-  position: relative;
-  outline-offset: 0.2rem;
-}
-
-.conversations-settings {
-  position: relative;
-
-  &--badge::after {
-    .dot-md;
-    position: absolute;
-    top: 10px;
-    right: 10px;
-    background-color: currentColor;
-    color: inherit;
-    content: '';
-  }
-}
-
```

Comment:
This is the style for the previous button and indicator badge and can be removed